mkdocs.yml

```diff
@@ -110,8 +110,17 @@ nav:
       - Create a privacy-enabled network: Tutorials/Create-Privacy-enabled-network.md
       - Send private transactions: Tutorials/Send-private-transaction.md
       - Create a permissioned network: Tutorials/CreatePermissionedNetwork.md
-      - Connect to a node: Tutorials/Connecting-to-a-node.md
       - Use a plugin: Tutorials/Use-Plugin.md
+  - Developing Applications:
```

Comment:
@joshuafernandes, would you be opposed to me moving this to How to > Develop applications?

Reply:
As discussed, you based this on user feedback, so I'll keep this as is.